src/config/app_config.rs

```diff
@@ -11,6 +11,7 @@ pub struct AppConfig {
     pub log_level: String,
     pub db: DatabaseConfig,
     pub jwt: JwtConfig,
+    pub tz_offset: i32
```

Comment:
I think server should return always UTC time no matter what server timezone is and timezone should be handled by client's local timezone.

Reply:
You're right, I'll remove it.